src/ios-deploy/ios-deploy.m

```diff
@@ -129,6 +129,7 @@
 char *symbols_download_directory = NULL;
 char *profile_uuid = NULL;
 char *profile_path = NULL;
+int pid = -1;
```

Comment:
naming this command_pid or something unique would make this easier to read since there are also local pid variables.